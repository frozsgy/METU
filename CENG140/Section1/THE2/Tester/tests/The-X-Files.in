ADD CIGARETTESMOKINGMAN
SUBSCRIBE CIGARETTESMOKINGMAN SYNDICATE
ADD KERSH
ADD SKINNER
ADD MULDER
ADD SCULLY
ADD KRYCEK
ADD LANGLEY
ADD FROHIKE
ADD BYERS
SUBSCRIBE FROHIKE CONSPIRACY
SUBSCRIBE LANGLEY CONSPIRACY
SUBSCRIBE BYERS CONSPIRACY
SUBSCRIBE KRYCEK SYNDICATE
SUBSCRIBE MULDER UFO
SUBSCRIBE MULDER ALIENS
SUBSCRIBE MULDER CONSPIRACY
SUBSCRIBE MULDER ABDUCTION
SUBSCRIBE MULDER XFILES
SEND SKINNER XFILES
SUBSCRIBE SKINNER XFILES
SEND KERSH XFILES
SUBSCRIBE SCULLY XFILES
SEND SCULLY XFILES
SEND CIGARETTESMOKINGMAN XFILES
UNSUBSCRIBE KRYCEK SYNDICATE
UNSUBSCRIBE SCULLY XFILES
SUBSCRIBE SCULLY ABDUCTION
SUBSCRIBE SKINNER CONSPIRACY
SEND KRYCEK ABDUCTION
SUBSCRIBE SCULLY XFILES
SUBSCRIBE SCULLY CANCER
SEND CIGARETTESMOKINGMAN CANCER
SEND SCULLY SYNDICATE
DELETEUSER MULDER
UNSUBSCRIBE SCULLY CANCER
ADD DOGGETT
ADD REYES
SUBSCRIBE DOGGETT XFILES
SUBSCRIBE REYES XFILES
SEND SKINNER XFILES
UNSUBSCRIBE SCULLY XFILES
ADD MULDER
SUBSCRIBE MULDER ALIENS
SUBSCRIBE MULDER UFO
SUBSCRIBE MULDER CONSPIRACY
SEND KERSH XFILES
SEND SKINNER CONSPIRACY
SEND MULDER CONSPIRACY
SEND REYES SYNDICATE
DELETEUSER FROHIKE
DELETEUSER BYERS
DELETEUSER LANGLEY
SEND KERSH XFILES
UNSUBSCRIBE REYES XFILES
UNSUBSCRIBE DOGGETT XFILES
SUBSCRIBE REYES SYNDICATE
SUBSCRIBE MULDER XFILES
SUBSCRIBE SCULLY XFILES
SEND SKINNER XFILES
