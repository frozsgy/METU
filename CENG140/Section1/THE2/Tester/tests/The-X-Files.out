1: 4 6 7 12
2: 2 9 13
3: 1 2 3 4 8 9 10 11 13 14
5: 5 7 14
6: 5
10:
11: 12
12: 10 11 14
